spec/localeapp/cli/pull_spec.rb

```diff
@@ -35,6 +35,21 @@
     end
   end
 
+  context "if a target directory is provided" do
+    let(:target_dir) { 'target/dir' }
+    before do
+      @puller.instance_variable_set(:@target_dir, target_dir)
+    end
+
+    it "calls the updater with a target_dir" do
```

Comment:
"with the target directory" I would say, here we do not care about a
specific variable name, and should not know it.